modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/exportimport/data/handler/StagedLayoutSetStagedModelDataHandler.java

```diff
@@ -501,6 +506,12 @@ private void _exportLayouts(
 
 		portletDataContext.getExportDataGroupElement(Layout.class);
 
+		if (!portletDataContext.isPrivateLayout() &&
```

Comment:
I also wonder if we should include there the next condition of the code:

`group.isLayoutPrototype()`

It seems for that use case, the list of Layouts is different. We might need to keep the previous flow for that use case. Could we confirm this with the Page Management team?